Update `cascade` default value to background

KimDoubleB opened the pull request:
The default value of `cascade` was changed to `background`, but there was a document that has not been modified yet, so I modified it.

Documents related to cascade
- https://kubernetes.io/docs/tasks/administer-cluster/use-cascading-deletion/#use-foreground-cascading-deletion
- https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#delete

sftim commented:
Usually when we revise articles, we leave the original text and insert a correction (eg in italics).

aaron-kasten commented:
@sftim @KimDoubleB Looks good!  I'm good with either leaving the original in (as it applies to pre `kubectl` <=1.20) or replacing the text as in this PR (as there probably aren't many pre 1.20 version of `kubectl` installed)

KimDoubleB commented:
@aaron-kasten I read your blog well. It really helped me a lot.

I know that the `cascade` option has changed since the kubectl 1.20.x version.
If you look at the context I modified, `--cascade=orphan` appears. That is, cascade option of kubectl 1.20.x later version is used. (This has been changed in this PR https://github.com/kubernetes/website/pull/29394)
However, `The default for cascade is true` is before kubectl 1.20.x. I thought there could be errors in various versions of the context., so I modified it.

reylejano commented on content/en/blog/_posts/2021-05-14-using-finalizers-to-control-deletion.md:
````diff
@@ -190,7 +190,7 @@ kubectl get configmap
 ```
 
-To sum things up, when there's an override owner reference from a child to a parent, deleting the parent deletes the children automatically. This is called `cascade`. The default for cascade is `true`, however, you can use the --cascade=orphan option for `kubectl delete` to delete an object and orphan its children. 
+To sum things up, when there's an override owner reference from a child to a parent, deleting the parent deletes the children automatically. This is called `cascade`. The default for cascade is `background`, however, you can use the --cascade=orphan option for `kubectl delete` to delete an object and orphan its children. 
````
```suggestion
To sum things up, when there's an override owner reference from a child to a parent, deleting the parent deletes the children automatically. This is called `cascade`. The default for cascade is `true`, however, you can use the --cascade=orphan option for `kubectl delete` to delete an object and orphan its children.  *Update: starting with kubectl v1.20, the default for cascade is `background`.*

```
How about something like this that keeps the original but adds an *Update* about changes starting with kubectl v1.20

KimDoubleB replied:
That's cool

reylejano commented:
/lgtm
